test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -383,6 +383,559 @@ class CharacterTest {
     assertFalse(Character.isDigit('\uFBFC'))
   }
 
+  @Test def toLowerCase_compare_char_and_codepoint(): Unit = {
+    for (ch <- Character.MIN_VALUE to Character.MAX_VALUE)
+      assertEquals(Character.toLowerCase(ch), Character.toLowerCase(ch.toInt).toChar)
+  }
+
+  @Test def toLowerCase_Char(): Unit = {
```

Comment:
Similarly to what I said for `toTitleCase`, this test is basically redundant with `toLowerCase_Int()`, now, so it can be removed.